generated/nidcpower/nidcpower.proto

```diff
@@ -43,7 +43,6 @@ service NiDCPower {
   rpc ConfigureDigitalEdgeSourceTriggerWithChannels(ConfigureDigitalEdgeSourceTriggerWithChannelsRequest) returns (ConfigureDigitalEdgeSourceTriggerWithChannelsResponse);
   rpc ConfigureDigitalEdgeStartTrigger(ConfigureDigitalEdgeStartTriggerRequest) returns (ConfigureDigitalEdgeStartTriggerResponse);
   rpc ConfigureDigitalEdgeStartTriggerWithChannels(ConfigureDigitalEdgeStartTriggerWithChannelsRequest) returns (ConfigureDigitalEdgeStartTriggerWithChannelsResponse);
-  rpc ConfigureLCRCompensation(ConfigureLCRCompensationRequest) returns (ConfigureLCRCompensationResponse);
```

Comment:
Looks like this would be a `breaking change` right?

Reply:
Hmm, yes and no. Compared to previous versions of this .proto in main -> Yes. Compared to previous versions of this .proto in the GitHub Releases -> No.

I'm wary of marking it a breaking-change because I think it'll be confusing in the next release notes. Thoughts?

Reply:
We haven't released it, so we shouldn't mark it breaking.